.gitignore

```diff
@@ -16,6 +16,7 @@ __pycache__
 cmake_build/
 .idea/**
 /build/
+[Bb]uild/
```

Comment:
Is this needed? Are there subdirectories called build generated as part of this? Or did you mean to just change the preceding line to /[Bb]uild/?

Reply:
@martinwicke seems /build/ doesn't work at all... I thought it's treated has a root folder rather than a relative path to `build`